Gemfile

```diff
@@ -21,9 +21,8 @@ rails_version = ENV["RAILS_VERSION"] || "default"
 
 rails = case rails_version
   when "default"
-    ">= 3.1.0"
```

Comment:
I mean, why enforce a default when `ENV["RAILS_VERSION"]` is not provided - cause I do get the point of this variable (testing across multiple Rails versions).